cluster/charts/crossplane/README.md

```diff
@@ -152,7 +152,7 @@ replicas: 1
 deploymentStrategy: RollingUpdate
 
 image:
-  repository: crossplane/crossplane
+  repository: xpkg.upbound.io/crossplane/crossplane
```

Comment:
This seems to be more than just style updates. Is this intentional?

Reply:
this was because it was branched off https://github.com/crossplane/crossplane/pull/4831

I tried to rebase and fix it and just made everything worse. I'm going to close and open a new one instead of fixing git.